controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -1806,5 +1913,63 @@ int cli_bml::get_dcs_scan_results(const std::string &radio_mac, unsigned int max
 {
     std::cout << "get_dcs_scan_results" << std::endl;
 
+    if (max_results_size == 0) {
+        std::cout << __func__ << "invalid input, max_results_size==0" << std::endl;
+        return -1;
+    }
+
+    struct BML_DCS_NEIGHBOR_AP *results = nullptr;
+
+    results =
+        (struct BML_DCS_NEIGHBOR_AP *)calloc(max_results_size, sizeof(struct BML_DCS_NEIGHBOR_AP));
+    if (results == nullptr) {
```

Comment:
Did you run KW on this? please refrain from using `if (ptr == nullptr)` and instead use `if (!ptr)` to avoid KW issues.

Reply:
removed the <calloc> entirely - and will run KW